DSharpPlus.Lavalink/EventArgs/GuildConnectionCreatedEventArgs - Copy.cs

```diff
@@ -21,7 +21,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
-using Emzi0767.Utilities;
+using DSharpPlus.AsyncEvents;
```

Comment:
- Copy.cs

Reply:
everyone noticed this, yet noone deleted this file :kekw:

Reply:
remove this file already